data/transactions/logic/opcodes.go

```diff
@@ -184,6 +189,36 @@ func opBranch() OpDetails {
 	return d
 }
 
+func multiOp(dispatch byte, name string, rootVersion uint64, children ...OpSpec) OpSpec {
+	ret := OpSpec{Opcode: dispatch, Name: name, OpDetails: opDefault()}
+	ret.childOps = children
+	ret.Version = rootVersion
```

Comment:
Why not build this all in the struct literal?

Reply:
Added the version into the literal, but the children I think have to come after unless we want to build up OpDefault() manually in the literal since the children go in OpDetails

Reply:
I meant that since you have the `children` here, why is the a literal on line 193, then you plug the other things in after, instead of one literal.

Reply:
I think I'm missing something, but how would we add children into the OpDetails if we also want to use opDefault without either having a function act on opDefault or manually writing all of opDefault

I could set a variable for OpDefault, then add children, then make the literal, but it would end up being the same number of lines

Reply:
Oh, sorry. I forgot that `children` was in the details, not in the spec, because visually, `ret.childOps` looks like it.  I feel that Go maybe should allow `childOps: children` in the struct literal. But it doesn't. OK!